addon/components/ui-button/component.ts

```diff
@@ -15,9 +17,10 @@ export default class UiButton extends Component {
 
   onClick?: Function | null = null;
   label?: string = "";
-  appearance?: string = "default"; //Enum: default, strong, minimal
-  intent?: string = "none"; //Enum: none, success, warning, danger
-  size?: string = "default"; //Enum: small, large
+  appearance?: "default" | "strong" | "minimal" = "default";
+  width?: string; //Accepts any standard CSS width value
```

Comment:
Needs a default value

Reply:
What is the benefit of having a default value here?

Reply:
Because the variable is optional, no default value means it could be `undefined` but `width` can only be a `string`. Surprised VSCode is not giving you a TS error here.

It either needs to be changed to 
```
width?: string | undefined;
```

or 

```
width? : string = "";
```

I prefer the latter :)